conbench/app/cluster.md

```diff
@@ -0,0 +1,9 @@
+# Cluster
```

Comment:
I could be wrong, but I think the mission is to display all the Conbench machines dynamically in the UI, rather than hardcoded in markdown.

I've merged a starting point here: https://github.com/conbench/conbench/pull/322/files

Let me know if you have any questions.

Cheers!

Reply:
Exactly! Thank you Diana

Reply:
There's still lots to do. (my machine name is `diana`)

<img width="1252" alt="Screen Shot 2022-03-30 at 9 52 24 PM" src="https://user-images.githubusercontent.com/878798/160973133-3f31f496-3038-49c5-a6a2-9e5bd62a0d5c.png">
<img width="1280" alt="Screen Shot 2022-03-30 at 9 52 32 PM" src="https://user-images.githubusercontent.com/878798/160973140-a8da103d-ef97-4608-8629-5297eae0649f.png">

Reply:
So glad to read your thought on this @dianaclarke . Thanks!

Reply:
A first glance .. 
![conbench_hardware_entity_1](https://user-images.githubusercontent.com/20308634/160980399-d0833c1e-5a2e-42c4-8f8f-49fc1e816d06.png)

Reply:
That's looking good!